lib/poloniex.js

```diff
@@ -513,201 +513,219 @@ class Poloniex extends EventEmitter {
 
   // WebSocket API
   openWebSocket(options) {
-    this.wsVersion = options && options.version === 2 && 2 || 1;
-    switch(this.wsVersion) {
-      case 1: {
-        if (this.ws) {
-          this.ws.close();
+    
+    this.returnTicker().then(currencies => {
```

Comment:
in case `returnTicker` returns an error, how should it behave?

Reply:
good call. let me look into it